package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "aqovia-vscode-extension-pack",
+  "displayName": "Aqovia VSCode ExtensionPack",
+  "description": "Aqovia Visual Studio Code Extension Pack",
+  "version": "1.3.34",
+  "publisher": "Aqovia",
+  "license": "SEE LICENSE IN LICENSE.txt",
+  "engines": {
+    "vscode": "^1.80.0"
+  },
+  "scripts": {
+    "pack": "vsce package --out VSCodeExtensionsPack.vsix"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Aqovia/VisualStudioCode.ExtensionPack"
+  },
+  "categories": [
+    "Extension Packs"
+  ],
+  "extensionPack": [
```

Comment:
I am somewhat torn as to whether we really should be installing installing all the language/framework-specific extensions here. This is supposed to be stuff that we want installed for everyone and should be as light and universal as possible. 

I think that ideally things like the Polymer IDE (which is deprecated btw, and only relevant for legacy projects - but we need to keep until we retire them) and the the Python Test Explorer should not be installed globally but instead added to [Recommended Extensions list for each individual workspace / repo](https://tech.serhatteker.com/post/2020-05/vscode-extensions-recommended/). This ensures you easily get the extensions you need when opening a project but without everyone having to install a heavy list that they mostly rarely use. It also works nicely with Github codespaces which will auto-install the recommended extensions.

On the other hand we have literally hundreds of repos that we would need to update to include their specific recommendations. In some cases we'd also be copy/pasting lists of multiple extensions - so we'd probably want to create some of our own language/framework-specific extension packs. 

Therefore, for now, I think we're doing the right thing by including everything but I think in the Readme we should make clear what the vision is so that we don't blindly expand this global list in future and evolve towards making more use of language/framework-specific packs and Recommendations inside Workspaces.

Reply:
Added the following text to the CONTRIBUTING file - "The intent of this extension pack is to provide an "all round" or "universal" set of extensions. For extensions particular to specific workspaces/repos, a recommended extensions list can be used as detailed here - https://tech.serhatteker.com/post/2020-05/vscode-extensions-recommended/"